### PR TITLE
An update regarding the change introduced in SOFA

### PR DESCRIPTION
**Pull request by Younesssss**

### CODE REVIEW

File: src/BeamAdapter/component/controller/SutureController.h [hunk 1]
```diff
@@ -18,266 +18,268 @@
...
+#include <SofaUserInteraction/MechanicalStateController.h>
+#include <SofaMeshCollision/PointModel.h>
+#include <SofaMeshCollision/LineModel.h>
+
+#include <BeamAdapter/component/WireBeamInterpolation.h>
+#include <BeamAdapter/config.h>
+#include <sofa/simulation/mechanicalvisitor/MechanicalPropagateOnlyPositionAndVelocityVisitor.h>
+#include <sofa/simulation/mechanicalvisitor/MechanicalProjectPositionAndVelocityVisitor.h>
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (epernod) [inline src/BeamAdapter/component/controller/SutureController.h:50]: Any idea why the whole file is marked as modified?

File: src/BeamAdapter/component/controller/AdaptiveBeamController.h [hunk 2]
```diff
@@ -51,9 +51,13 @@
...
-            template <class T>
-            class EdgeSetGeometryAlgorithms;
-            class EdgeSetTopologyModifier;
+            namespace container {
+                namespace dynamic {
+                    template <class T>
+                    class EdgeSetGeometryAlgorithms;
+                    class EdgeSetTopologyModifier;
+                }
+            }
```
Comment (epernod) [inline src/BeamAdapter/component/controller/AdaptiveBeamController.h:60]:
```suggestion
namespace sofa::component::topology::container::dynamic 
{
    template <class T>
    class EdgeSetGeometryAlgorithms;
    class EdgeSetTopologyModifier;
}
```

File: src/BeamAdapter/component/controller/AdaptiveBeamController.h [hunk 3]
```diff
@@ -51,9 +51,13 @@
 namespace sofa {
     namespace component {
         namespace topology {
```
Comment (epernod) [inline src/BeamAdapter/component/controller/AdaptiveBeamController.h:53]:
```suggestion
```

File: src/BeamAdapter/component/controller/AdaptiveBeamController.h [hunk 4]
```diff
@@ -51,9 +51,13 @@
...
+            namespace container {
+                namespace dynamic {
+                    template <class T>
+                    class EdgeSetGeometryAlgorithms;
+                    class EdgeSetTopologyModifier;
+                }
+            }
         }
     }
 }
```
Comment (epernod) [inline src/BeamAdapter/component/controller/AdaptiveBeamController.h:63]:
```suggestion
```

**Comment by damienmarchal** [comment]
Thanks for the PR... will be superceed by PR #11

**Pull request closed without merging**